pyproject.toml

```diff
@@ -71,49 +71,161 @@ line-length = 79
 indent-width = 4
 
 target-version = "py38"
+exclude = [
+  "CSV",
+  "ref.*?py",
+]
 
 [tool.ruff.lint]
-fixable = ["ALL"]
-preview = true
+fixable = []
+# preview = true
+# logger-objects = []
+task-tags = ["TODO", "FIXME", "XXX"]
 select = [
+  "A", # flake8-builtins
+  "ARG", # flake8-unused-arguments
+  "COM", # flake8-commas
+  "D", # pydocstyle
   "E", # pycodestyle
+  "E701", # multiple-statements-on-one-line-colon
+  "E722", # bare-except
   "F", # Pyflakes
+  "F401", # unused-import
+  "G", # flake8-logging-format
+  "I", # isort
+  "N", # pep8-naming
+  "NPY", # numpy-specific
+  "PL", # Pylint
+  "Q", # flake8-quotes
+  "UP", # pyupgrade
   "W", # Warning
   ]
 
 ignore = [
-  "B008", # function-call-in-default-argument
-  "B024", # abstract-base-class-without-abstract-method
-  "B028", # no-explicit-stacklevel
-  "B905", # zip-without-explicit-strict
+  # Currently don't conform but we might want to reconsider
+  "A001", # builtin-variable-shadowing
+  # Currently don't conform but we might want to reconsider
+  "A002", # builtin-argument-shadowing
+  "ARG002", # unused-method-argument
+  "ARG003", # unused-class-method-argument
+  "ARG004", # unused-static-method-argument
+  "ARG005", # unused-lambda-argument
+#  "B008", # function-call-in-default-argument
+#  "B024", # abstract-base-class-without-abstract-method
+#  "B028", # no-explicit-stacklevel
+#  "B905", # zip-without-explicit-strict
+  "COM812", # missing-trailing-comma
   "D105", # undocumented-magic-method
+  "D200", # fits-on-one-line
   "D202", # no-blank-line-after-function
+  "D204", # one-blank-line-after-class
   "D205", # blank-line-after-summary
+  "D207", # under-indentation
   "D209", # new-line-after-last-paragraph
-  "D212", # multi-line-summary-first-line
-  "D213", # multi-line-summary-second-linek
+#  "D212", # multi-line-summary-first-line
```

Comment:
Why did you comment these ones out?

Reply:
I've also been using ruff from the beginning, and slowly changing the rules to conform with the sup3r standard as I get errors. At some point `D212` was a requirement. Yesterday I removed everything and checked each one. As it is, sup3r pass without ignoring `D212`. I left commented while we double check everything with the intention to remove soon.